### PR TITLE
Coupons That are Not Valid Show In The Plugin

### PR DESCRIPTION
**Pull request by nldn**
https://app.asana.com/0/1137432207089210/1137381555248402

### CODE REVIEW

File: widget/controllers/widget.home.controller.js [hunk 1]
```diff
@@ -32,13 +32,23 @@
...
           limit: PAGINATION.itemCount,
           filter: {
-            "$or": [{
-              "$json.expiresOn": {$gte: WidgetHome.yesterdayDate}
-            }, {"$json.expiresOn": ""}]
+            "$and": [
+              {
+                "$or": [{
+                  "$json.expiresOn": { $gte: WidgetHome.yesterdayDate }
+                }, { "$json.expiresOn": "" }]
```
Comment (mikailbayram) [inline widget/controllers/widget.home.controller.js]: Why are we filtering empty strings? @nldn 
Same question goes for each filter you added

Comment (nldn) [reply widget/controllers/widget.home.controller.js]: This filters the case if attribute is not set.

**Comment by mikailbayram** [comment]
@najiatallah good to go

File: control/content/index.html [hunk 2]
```diff
@@ -87,7 +87,7 @@
     }
 
     .pac-container {
```
Comment (najiatallah) [inline control/content/index.html:89]: Not a big fan of hardcoded values

**Comment by najiatallah** [comment]
Approved

**Pull request merged**